2.TransformStreams/transformStreams.js

```diff
@@ -0,0 +1,17 @@
+import path from "path";
+import __dirname from "../path.js";
+import { Transform } from "stream";
+import fs from "fs";
+
+const data = fs.createReadStream(path.join(__dirname, "2.TransformStreams", "text.json"));
+const transformFile = fs.createWriteStream(path.join(__dirname, "2.TransformStreams", "transform.json"));
+
+const transform = new Transform({
+  transform(chunk, encoding, callback) {
```

Comment:
In order to work with other JS types such as objects, you can use `objectMode:true` config and work with chunk as JSON, and using parse and stringify to add new item.